packages/web-app-files/src/components/SideBar/SideBar.vue

```diff
@@ -234,14 +234,6 @@ export default defineComponent({
             this.highlightedFile.webDavPath,
             DavProperties.Default
           )
-          if (
-            this.hasShareJail &&
-            (isLocationSharesActive(this.$router, 'files-shares-with-me') ||
-              (isLocationSpacesActive(this.$router, 'files-spaces-share') &&
-                this.highlightedFile.path === '/'))
-          ) {
-            item.name = this.highlightedFile.name
-          }
```

Comment:
I remember there was a reason why I added this, although I don't recall which one exactly. I think it has something to do with the name display on re-shares. Let me check.

Reply:
Afair you added it before you implemented fetching the name webdav prop, so should be a leftover and not needed anymore. But let's double check!

Reply:
You're right, the sidebar seems to be fine for re-shares 👍